src/components/Home/Add Item/AddItemForm/index.js

```diff
@@ -0,0 +1,212 @@
+                                        <button className="btn btn-link addPhotoButton">
+                                            Add Photo
+                                    </button>
+                                    </form>
+                                    <div className="modal-footer">
+                                        <button type="button" className="btn btn-secondary" data-dismiss="modal">Close</button>
+                                        <button type="button" className="btn btn-primary" onClick={this.addButtonPressed}>Add Item</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                )
+            case "Chores":
+                return (
+                    <div className="modal fade" id={this.props.type} tabIndex="-1" role="dialog" aria-labelledby="myModalLabel">
+                        <div className="modal-dialog" role="document">
+                            <div className="modal-content">
+                                <div className="modal-header">
+                                    <h4 className="modal-title" id="myModalLabel">Add {this.props.type}</h4>
+                                    <button type="button" className="close" data-dismiss="modal" aria-label="Close">&times;</button>
+                                </div>
+                                <div className="modal-body">
+                                    <form onSubmit={this.addButtonPressed}>
+                                        <div className="form-group">
+                                            <div className="form-row">
+                                                <label className="modal-subhead col-7 mr-3 ml-1" htmlFor={"title" + this.props.type}>Title</label>
+                                            </div>
+                                            <div className="form-row">
+                                                <input type="text" className="form-control col-7 mr-3 ml-1" id={"title" + this.props.type} aria-describedby="Title Help" placeholder="Enter title" required></input>
+                                            </div>
+
+                                        </div>
+                                        <button className="btn btn-link addPhotoButton">
+                                            Add Photo
+                                    </button>
+                                        <div className="form-group">
+                                            <div className="form-group">
+                                                <label className="modal-subhead" htmlFor="exampleInputEmail1">Deadline Date</label>
+                                                <div className="calendarGroup">
+                                                    {<Calendar minDate={new Date()} onClickDay={this.handleChoreCalendarChange} />}
```

Comment:
Where does this value for date go? I want to be able to access it on the home page to put on the main calendar

Reply:
So that date just sets the calendar to set the minimum date that can be clicked to the current date.